superbench/benchmarks/micro_benchmarks/cudnn_test/cudnn_benchmark.h

```diff
@@ -0,0 +1,254 @@
+// Copyright(c) Microsoft Corporation.
```

Comment:
The file name is cudnn_benchmark.h, but it's not related to it. Why not merge this with cudnn_fucntion.h?

Reply:
splited them by class